src/rdo.rs

```diff
@@ -465,97 +465,97 @@ fn luma_chroma_mode_rdo<T: Pixel> (luma_mode: PredictionMode,
   luma_mode_is_intra: bool,
   mode_context: usize,
   mv_stack: &ArrayVec<[CandidateMV; 9]>) {
-    let (tx_size, mut tx_type) = rdo_tx_size_type(
-      fi, ts, cw, bsize, tile_bo, luma_mode, ref_frames, mvs, false,
-    );
+  let (tx_size, mut tx_type) = rdo_tx_size_type(
```

Comment:
Why is the indentation gone?

Reply:
It was erroneous - indented one level too far.

Reply:
Unless the GitHub view is wrong, it looks like there is no indentation in the inner scope of the function now.

Reply:
Unless the GitHub "Files changed" view is wrong, it looks like there is no indentation in the inner scope of the function now.

Reply:
I see, the argument list indentation made it appear as if the body was not indented properly in the diff view.